templates/circle-agent/src/lib/WalletService.ts

```
@@ -0,0 +1,208 @@
+    }
+  }
+
+  static async transfer(
+    fromWallet: WalletData,
+    destinationAddress: string,
+    amount: string,
+    tokenSymbol: string = 'USDC' // Default to USDC but allow other tokens
+  ): Promise<TransactionStatus> {
+    try {
+      // Get token balances to find the requested token
+      const balances = await this.checkBalance(fromWallet);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)